src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystemAliases.cs

```diff
@@ -0,0 +1,114 @@
+        /// eg. <code>http://myserver:8080/tfs/defaultcollection/myproject/_git/myrepository</code>.</param>
+        /// <param name="pullRequestId">ID of the pull request.</param>
+        /// <returns>Object for writing issues to Team Foundation Server or Visual Studio Team Services pull request.</returns>
+        /// <example>
+        /// <para>Report code analysis issues reported as MsBuild warnings to a TFS pull request:</para>
+        /// <code>
+        /// <![CDATA[
+        ///     ReportCodeAnalysisIssuesToPullRequest(
+        ///         MsBuildCodeAnalysis(@"C:\build\msbuild.log", MsBuildXmlFileLoggerFormat),
+        ///         TfsPullRequests(new Uri("http://myserver:8080/tfs/defaultcollection/myproject/_git/myrepository"), 5));
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory(CakeAliasConstants.PullRequestSystemCakeAliasCategory)]
+        public static IPullRequestSystem TfsPullRequests(
+            this ICakeContext context,
+            Uri repositoryUrl,
+            int pullRequestId)
+        {
+            context.NotNull(nameof(context));
+            repositoryUrl.NotNull(nameof(repositoryUrl));
+
+            return context.TfsPullRequests(new TfsPullRequestSettings(repositoryUrl, pullRequestId));
+        }
+
+        /// <summary>
+        /// Gets an object for writing issues to Team Foundation Server or Visual Studio Team Services pull request
+        /// using the specified settings.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="settings">Settings for accessing the pull request system.</param>
+        /// <returns>Object for writing issues to Team Foundation Server or Visual Studio Team Services pull request.</returns>
+        /// <example>
+        /// <para>Report code analysis issues reported as MsBuild warnings to a TFS pull request:</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var pullRequestSettings =
+        ///         new TfsPullRequestSettings(
+        ///             new Uri("http://myserver:8080/tfs/defaultcollection/myproject/_git/myrepository"),
+        ///             "refs/heads/feature/myfeature");
+        ///
+        ///     ReportCodeAnalysisIssuesToPullRequest(
+        ///         MsBuildCodeAnalysis(@"C:\build\msbuild.log", MsBuildXmlFileLoggerFormat),
+        ///         TfsPullRequests(pullRequestSettings));
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory(CakeAliasConstants.PullRequestSystemCakeAliasCategory)]
+        public static IPullRequestSystem TfsPullRequests(
+            this ICakeContext context,
+            TfsPullRequestSettings settings)
+        {
+            context.NotNull(nameof(context));
+            settings.NotNull(nameof(settings));
+
+            return new TfsPullRequestSystem(context.Log, settings);
```

Comment:
Connection is now created on demand. And created https://github.com/cake-contrib/Cake.Prca/issues/7 for changing signature of `MarkThreadAsFixed`